File: features/snippets/external_storage/threshold/threshold_config.go
package threshold

import (
	"log"

	"go.temporal.io/sdk/client"
	"go.temporal.io/sdk/converter"
)

func ThresholdConfig(driver converter.StorageDriver) {
	// @@@SNIPSTART go-external-storage-threshold
	c, err := client.Dial(client.Options{
		ExternalStorage: converter.ExternalStorage{
			Drivers:              []converter.StorageDriver{driver},
			PayloadSizeThreshold: 1,
		},
	})
	// @@@SNIPEND
	if err != nil {
		log.Fatalf("connect to Temporal: %v", err)
	}
	defer c.Close()
}

File: features/snippets/external_storage/threshold/threshold_config.java
import io.temporal.payload.storage.ExternalStorage;
import io.temporal.payload.storage.StorageDriver;

class ThresholdConfigSnippet {
  static ExternalStorage configure(StorageDriver driver) {
    // @@@SNIPSTART java-external-storage-threshold
    return ExternalStorage.newBuilder().setDriver(driver).setPayloadSizeThreshold(0).build();
    // @@@SNIPEND
  }
}
